src/renderer/components/side-nav-more-options/side-nav-more-options.vue

```diff
@@ -47,9 +57,13 @@
       <div
         v-if="!hideTrendingVideos"
         class="navOption"
+        role="link"
```

Comment:
you could make these into actual router links (see side-nav component), that way other behaviour like control clicking, middle clicking, and the "Open in a New Window" context menu will all automatically work.